app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -122,6 +126,53 @@
         $node.attr("aria-expanded", state);
       }
 
+      function setSessionStorage() {
+        var isOpenSubsections = $('.subsection--is-open').length;
+        if (isOpenSubsections) {
+          var $openSubsections = $('.subsection--is-open');
+          $openSubsections.each(function(index) {
+            var subsectionOpenContentId = $(this).find('.subsection__content').attr('id');
+            sessionStorage.setItem( subsectionOpenContentId , subsectionOpenContentId);
+          });
+        }
+      }
+
+      function removeSessionStorage() {
```

Comment:
I think I'd find this `removeSessionStorage()` and the above `setSessionStorage()` function names more representative of the role they have if they were called `setOpenTopicSections()` and `removeClosedTopicSections()`? Or `registerOpenTopicSections()` and `unregisterClosedTopicSections()`? Not sure.. Something like that maybe.